ui/components/DataTable/TableView/TableView.tsx

```diff
@@ -0,0 +1,57 @@
+import { Table, TableContainer } from "@material-ui/core";
+import React from "react";
+import TableBodyView from "./TableBody";
+import TableHeader from "./TableHeader";
+import { TableViewProps } from "./types";
+
+const TableView = ({
+  rows,
+  fields,
+  id,
+  checkedFields,
+  defaultSortedField,
+  onSortChange,
+  hasCheckboxes,
+  onBatchCheck,
+}: TableViewProps) => {
+  const onCheckChange = (checked: boolean, id: string) => {
+    const chk = [...checkedFields];
+    if (checked) {
+      chk.push(id);
+    } else {
+      chk.splice(chk.indexOf(id), 1);
+    }
+    onBatchCheck(chk);
+  };
+  const onHeaderCheckChange = (checked: boolean) => {
+    if (checked) {
+      checkedFields = rows.map((r) => r.uid);
+    } else {
+      checkedFields = [];
+    }
+    onBatchCheck(checkedFields);
+  };
+  return (
+    <TableContainer id={id}>
+      <Table aria-label="simple table">
```

Comment:
❔  Is this `aria-label` informative to the user? It's a "simple table" for developers (comparing to the old all in one table), but for users it's just a table. And do we need an aria-label here at all?

[MDN](From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label):
> Note: While aria-label is allowed on any element that can have an accessible name, in practice, aria-label is only supported on interactive elements, widgets, landmarks, images and iframes.

Is the whole table considered an interactive element?

Maybe we could add some roles instead or in addition to `aria-label`?
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/table_role

Reply:
I'll remove the `aria-label`